tests/link/pytorch/test_basic.py

```diff
@@ -301,3 +302,16 @@ def test_pytorch_MakeVector():
     x_fg = FunctionGraph([], [x])
 
     compare_pytorch_and_py(x_fg, [])
+
+
+def test_pytorch_ifelse():
+    true_vals = np.r_[1, 2, 3]
+    false_vals = np.r_[-1, -2, -3]
+
+    for test_value, cond in [(0.2, 0.5), (0.5, 0.4)]:
+        a = scalar("a")
+        a.tag.test_value = np.array(test_value, dtype=config.floatX)
```

Comment:
No need for test values in the tag, just pass them directly to `compare_pytorch_and_py`